ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx

```diff
@@ -149,7 +149,7 @@ class ConfirmPaymentMethodScreen extends React.Component<Props, never> {
           <PaymentBannerComponent
             currentAmount={currentAmount}
             paymentReason={paymentReason}
-            recipient={recipient || UNKNOWN_RECIPIENT}
+            recipient={recipient || UNKNOWN_RECIPIENT_EMPTY}
```

Comment:
Can't we just make "recipient" prop optional and check it in PaymentBannerComponent?

Reply:
@fpersico PaymentBannerComponent is used in different part of the app (3 usage)
I make this change only for ConfirmPaymentMethodScreen as requested.
I don't know if in the other parts we want the same behaviour

If you think so I can change the code as suggested

Reply:
I think the behaviour must be the same in all the screen. What you think @matteodesanti?

Reply:
@Undermaken

> Can't we just make "recipient" prop optional and check it in PaymentBannerComponent?